packaging/nuget/package.ps1

```diff
@@ -0,0 +1,26 @@
+Push-Location $PSScriptRoot
+[Environment]::CurrentDirectory = $PWD
+
+$ProjectsDir = "..\..\src"
```

Comment:
What's the point of adding 2 lines of magic code which substitute 2 lines of obvious code?